src/tests/LoginTest.java

```diff
@@ -0,0 +1,44 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class LoginTest extends BaseTest{
+
+    @BeforeMethod
+    public void beforeTest(){
+        HomePage home = new HomePage(driver);
+        home.openHomePage()
+            .goToSignPage();
+    }
+
+    @Test(groups = {"Smoke", "Regression"})
+    public void pageElement(){
+        LoginPage page = new LoginPage(driver);
```

Comment:
You can move this variable to the class variable  and then use this construction "loginPage = homePage.openHomePage().goToSignPage();" in "BeforeTest"

Reply:
@vladimirKorzhukov
I already did try did and I keep getting "Test not found" error.